frame/bounties/src/lib.rs

```diff
@@ -385,10 +469,8 @@ decl_module! {
 		/// - O(1).
 		/// # </weight>
 		#[weight = <T as Config>::WeightInfo::unassign_curator()]
-		fn unassign_curator(
-			origin,
-			#[compact] bounty_id: BountyIndex,
-		) {
+		fn unassign_curator(origin, #[compact] bounty_id: BountyIndex)
+		{
```

Comment:
keep `{` location consistent, i.e. don't make a new line for it

Reply:
Sure, corrected it.